### ISSUE TITLE
set scan interval

### ISSUE THREAD

**Issue by tillmannschatz** [mKeRix/room-assistant#13]
Hey great tool! I was wondering, if you can set the timeinterval for the scans manually. For example just check every 20 seconds if the ibeacon is present.

Thanks!

**Comment by mKeRix** [comment]
Hey there,

since the beacons are not polled for but rather sent when their advertisement packets arrive there is no time interval. **However** you're in luck, somebody already submitted a merge request that does exactly what you want at #12. :)

It's already merged into `dev`. Sadly I didn't have time to test do the last tasks before including it in a new release last weekend, but I'll be sure to get on that this weekend. I'll let you know and close this issue once that is done.

**Comment by mKeRix** [comment]
I just merged everything into master, so you should be good to go. IF there are any problems feel free to reopen the ticket!
